From 098ea461478192bdce1c8daf9dced8b2edba861b Mon Sep 17 00:00:00 2001
From: Martijn Braam <martijn@brixit.nl>
Date: Wed, 7 Oct 2020 17:33:43 +0200
Subject: media: ov5640: set default ae target lower

The OV5640 tries to overexpose all photos by about 1 stop. This makes
the exposure target one stop lower.
---
 drivers/media/i2c/ov5640.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/media/i2c/ov5640.c b/drivers/media/i2c/ov5640.c
index 0e98ed8b3e08..4e0803484fd4 100644
--- a/drivers/media/i2c/ov5640.c
+++ b/drivers/media/i2c/ov5640.c
@@ -3875,7 +3875,7 @@ static int ov5640_probe(struct i2c_client *client)
 	sensor->current_link_freq =
 		ov5640_csi2_link_freqs[OV5640_DEFAULT_LINK_FREQ];
 
-	sensor->ae_target = 52;
+	sensor->ae_target = 28;
 
 	endpoint = fwnode_graph_get_next_endpoint(dev_fwnode(&client->dev),
 						  NULL);
-- 
2.35.3

